appveyor.yml

```diff
@@ -0,0 +1,110 @@
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\appveyor\\run_with_env.cmd"
+
+  matrix:
+
+    # Pre-installed Python versions, which Appveyor may upgrade to
+    # a later point release.
+    # See: http://www.appveyor.com/docs/installed-software#python
+
+    # Python 2.6.6 is the latest Python 2.6 with a Windows installer
+    # Python 2.6.9 is the overall latest
+    # https://www.python.org/ftp/python/2.6.6/
+    - PYTHON: "C:\\Python26"
+      PYTHON_VERSION: "2.6.6"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python26-x64"
+      PYTHON_VERSION: "2.6.6"
+      PYTHON_ARCH: "64"
+
+    # Python 2.7.10 is the latest Python 2.7 with a Windows installer
+    # Python 2.7.10 is the overall latest
+    # https://www.python.org/ftp/python/2.7.10/
+    - PYTHON: "C:\\Python27"
+      PYTHON_VERSION: "2.7.10"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python27-x64"
+      PYTHON_VERSION: "2.7.10"
+      PYTHON_ARCH: "64"
+
+    # Python 3.3.5 is the latest Python 3.3 with a Windows installer
+    # Python 3.3.6 is the overall latest
+    # https://www.python.org/ftp/python/3.3.5/
+    - PYTHON: "C:\\Python33"
+      PYTHON_VERSION: "3.3.5"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python33-x64"
+      PYTHON_VERSION: "3.3.5"
+      PYTHON_ARCH: "64"
+
+    # Python 3.4.3 is the latest Python 3.4 with a Windows installer
+    # Python 3.4.3 is the overall latest
+    # https://www.python.org/ftp/python/3.4.3/
+    - PYTHON: "C:\\Python34"
+      PYTHON_VERSION: "3.4.3"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python34-x64"
+      PYTHON_VERSION: "3.4.3"
+      PYTHON_ARCH: "64"
+
+install:
+  - ECHO "Filesystem root:"
+  - ps: "ls \"C:/\""
+
+  - ECHO "Installed SDKs:"
+  - ps: "ls \"C:/Program Files/Microsoft SDKs/Windows\""
+
+  # Install Python (from the official .msi of http://python.org) and pip when
+  # not already installed.
+  - ps: if (-not(Test-Path($env:PYTHON))) { & appveyor\install.ps1 }
+
+  # Prepend newly installed Python to the PATH of this build (this cannot be
+  # done from inside the powershell script as it would require to restart
+  # the parent CMD process).
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+
+  # Check that we have the expected version and architecture for Python
+  - "python --version"
+  - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
+
+  # Upgrade to the latest version of pip to avoid it displaying warnings
+  # about it being out of date.
+  - "pip install --disable-pip-version-check --user --upgrade pip"
+
+  # Install the build dependencies of the project. If some dependencies contain
+  # compiled extensions and are not provided as pre-built wheel packages,
+  # pip will build them from source using the MSVC compiler matching the
+  # target Python version and architecture
+  - "%CMD_IN_ENV% pip install wheel"
+  - "%CMD_IN_ENV% pip install nose"
+  - "%CMD_IN_ENV% pip install unittest2"
+  - "%CMD_IN_ENV% pip install cryptography"
```

Comment:
@tseaver I don't know why but `setup.py build` was [failing][1] when it tried to install `cryptography` (transitive dep of `pyOpenSSL`) but it successfully installs it here.

[1]: https://ci.appveyor.com/project/dhermes/gcloud-python/build/1.0.45.master/job/u7v2kf2ppak8rean